code/scripting/api/objs/enums.h

```diff
@@ -10,85 +10,88 @@
+const int32_t LE_ORDER_STAY_STILL            = 43;
+const int32_t LE_ORDER_UNDOCK                = 44;
+const int32_t LE_ORDER_WAYPOINTS             = 45;
+const int32_t LE_ORDER_WAYPOINTS_ONCE        = 46;
+const int32_t LE_ORDER_ATTACK_WING           = 69;
+const int32_t LE_ORDER_GUARD_WING            = 70;
+const int32_t LE_ORDER_ATTACK_SHIP_CLASS     = 74;
+const int32_t LE_PARTICLE_DEBUG              = 4;
+const int32_t LE_PARTICLE_BITMAP             = 5;
+const int32_t LE_PARTICLE_FIRE               = 6;
+const int32_t LE_PARTICLE_SMOKE              = 7;
+const int32_t LE_PARTICLE_SMOKE2             = 8;
+const int32_t LE_PARTICLE_PERSISTENT_BITMAP  = 9;
+const int32_t LE_SEXPVAR_CAMPAIGN_PERSISTENT = 22;
+const int32_t LE_SEXPVAR_NOT_PERSISTENT      = 23;
+const int32_t LE_SEXPVAR_PLAYER_PERSISTENT   = 24;
+const int32_t LE_SEXPVAR_TYPE_NUMBER         = 25;
+const int32_t LE_SEXPVAR_TYPE_STRING         = 26;
+const int32_t LE_TEXTURE_STATIC              = 10;
+const int32_t LE_TEXTURE_DYNAMIC             = 11;
+const int32_t LE_LOCK                        = 12;
+const int32_t LE_UNLOCK                      = 13;
+const int32_t LE_NONE                        = 15;
+const int32_t LE_SHIELD_FRONT                = 16;
+const int32_t LE_SHIELD_LEFT                 = 17;
+const int32_t LE_SHIELD_RIGHT                = 18;
+const int32_t LE_SHIELD_BACK                 = 19;
+const int32_t LE_MISSION_REPEAT              = 47;
+const int32_t LE_NORMAL_CONTROLS             = 48;
+const int32_t LE_LUA_STEERING_CONTROLS       = 49;
+const int32_t LE_LUA_FULL_CONTROLS           = 50;
+const int32_t LE_NORMAL_BUTTON_CONTROLS      = 51;
+const int32_t LE_LUA_ADDITIVE_BUTTON_CONTROL = 52;
+const int32_t LE_LUA_OVERRIDE_BUTTON_CONTROL = 53;
+const int32_t LE_VM_INTERNAL                 = 54;
+const int32_t LE_VM_EXTERNAL                 = 55;
+const int32_t LE_VM_TRACK                    = 56;
+const int32_t LE_VM_DEAD_VIEW                = 57;
+const int32_t LE_VM_CHASE                    = 58;
+const int32_t LE_VM_OTHER_SHIP               = 59;
+const int32_t LE_VM_EXTERNAL_CAMERA_LOCKED   = 60;
+const int32_t LE_VM_CAMERA_LOCKED            = 75;
+const int32_t LE_VM_WARP_CHASE               = 61;
+const int32_t LE_VM_PADLOCK_UP               = 62;
+const int32_t LE_VM_PADLOCK_REAR             = 63;
+const int32_t LE_VM_PADLOCK_LEFT             = 64;
+const int32_t LE_VM_PADLOCK_RIGHT            = 65;
+const int32_t LE_VM_WARPIN_ANCHOR            = 66;
+const int32_t LE_VM_TOPDOWN                  = 67;
+const int32_t LE_VM_FREECAMERA               = 68;
+const int32_t LE_VM_CENTERING                = 76;
+const int32_t LE_MESSAGE_PRIORITY_LOW        = 71;
+const int32_t LE_MESSAGE_PRIORITY_NORMAL     = 72;
+const int32_t LE_MESSAGE_PRIORITY_HIGH       = 73;
+const int32_t LE_OPTION_TYPE_SELECTION       = 77;
+const int32_t LE_OPTION_TYPE_RANGE           = 78;
+
+const int ENUM_NEXT_INDEX = 79; // <<<<<<<<<<<<<<<<<<<<<<
```

Comment:
This should be kept because new values are not necessarily added to the end of the list.

It would be nice to have some sort of compile-time check for duplicate values but I'm not sure how that could be done.

Reply:
I'm not sure what you are referring to here since nothing was removed here.
However, I just saw that I accidentally reused the value `77` so I'll need to change that.

Reply:
I must have misread the change; I thought `ENUM_NEXT_INDEX` was completely removed in the commit.  What you have looks good.